### PR TITLE
Dicegame1

### PR DESCRIPTION
**Pull request by JoeFoster-cn**

### CODE REVIEW

File: dicegame/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        <p id="player">Player 1</p>
+        <p id="score">0</p>
+            
+        <div class="dice">
+            <img id="diceImg" src="" width="25%" height="25%">
+        </div> 
+        <br>
+        <br>
+
+
```
Comment (tamjankowska) [inline dicegame/index.html]: You have some whitespace here you could get rid of.

File: dicegame/script.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+const result = document.querySelector("#player");
+const dice = document.querySelector(".dice");
+
+//default score
+let rollScore = 0;
+
+//roll click event, diceroll generates number, rollscore 0 becomes 0 + the diceroll
+roll.addEventListener("click", (event) => {
+    const diceRoll = Math.floor(Math.random()*6) + 1;
+    rollScore = rollScore + diceRoll
```
Comment (tamjankowska) [inline dicegame/script.js]: You could perhaps use `+=` here.

File: dicegame/script.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        result.innerHTML = "winner";
+        score.innerHTML = rollScore; //this is the score
+        restart.style.display = "block"; //start again button
+    }
+ //if diceroll hits 1 while score is less than 20, you lose
+    else if (diceRoll == 1 && rollScore < 20) {
+        result.innerHTML = "You lost!"; //you lose
+        score.innerHTML = rollScore; //shows score
+        restart.style.display = "block"; //start again button
+
```
Comment (tamjankowska) [inline dicegame/script.js]: Whitespace.

File: dicegame/script.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        score.innerHTML = rollScore; //shows score
+        restart.style.display = "block"; //start again button
+        
+    }
+//else show player 1, their score = to rollscore,  
+    else {
+        result.innerHTML = "Player 1";
+        score.innerHTML = rollScore;
+
+
```
Comment (tamjankowska) [inline dicegame/script.js]: Whitespace! :)

File: dicegame/script.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    }
+    let diceNum = document.getElementById("diceImg").src = "./img/dice" + diceRoll + ".png";
+    diceNum = document.querySelector("#diceImg");
+    diceNum.style.display = "block"; 
+})
+
+
```
Comment (tamjankowska) [inline dicegame/script.js]: You know what I'm going to say here...

**Review by tamjankowska — APPROVED** [review]
Bit of tidying up but looks great. I really like the comments to help me read through what your code is doing.

File: dicegame/script.js (hunk 5 above)
Comment (JoeFoster-cn) [reply dicegame/script.js]: Thanks Tam! :)

**Pull request merged**